packages/web-components/src/theme/set-theme.ts

```diff
@@ -10,12 +9,12 @@ const tokenNames = Object.keys(tokens) as (keyof Theme)[];
  */
 export const setTheme = (theme: Theme) => {
```

Comment:
The great thing about this now, is that we don't need to use this function at all if we can set the tokens we care about in the HTML document directly, since everything uses platform. Thank you!

Reply:
Was that needed previously?